lib/ocamlorg_web/graphql.ml

```diff
@@ -0,0 +1,330 @@
+open Ocamlorg
+
+type package_dep =
+  { name : string
+  ; constraints : string option
+  }
+
+type package_owner =
+  { full_name : string
+  ; handle : string option
+  ; image : string option
+  }
+
+type package_url =
```

Comment:
I believe this type can be removed now, I think it was left over from when you were converting the `Package.Info.url` type but now we are using it directly.

Reply:
We should add an `mli` file so that unused variables and types are caught at compile time